src/Stream.ts

```diff
@@ -61,6 +61,42 @@ export interface Emitter<A> {
   end(): void
 }
 
+export const fromAsyncIterable = <A>(iterable: AsyncIterable<A>): Fx.Fx<Async.Async | Stream<A>, void> => Fx.bracket(
+  Fx.sync(() => iterable[Symbol.asyncIterator]()),
+  iterator => Async.run(() => (iterator.return?.() ?? Promise.resolve()).then(() => { })),
```

Comment:
This was a little odd at first glance, but I had to add the `then` because `iterator.return()` doesn't return `void`

Reply:
What about moving the `then` in to the return:

```suggestion
  iterator => Async.run(() => (iterator.return?.() .then(() => {}) ?? Promise.resolve())),
```

Reply:
Fixed https://github.com/briancavalier/fx/pull/5/commits/8d5bac8640084704a004884cbe382b46112df6e7